lib/ansible/config/base.yml

```diff
@@ -1366,6 +1366,16 @@ GALAXY_COLLECTION_SKELETON_IGNORE:
   ini:
   - {key: collection_skeleton_ignore, section: galaxy}
   type: list
+GALAXY_COLLECTIONS_PATH_WARNING:
+  name: "ansible-galaxy collection install colections path warnings"
+  descriptions: "whether ``ansible-galaxy collection install`` should warn about ``--collection-path`` missing from configured ``collections_paths``"
```

Comment:
I *think* you can link to the COLLECTIONS_PATHS config by using :ref:\`collections_paths\` (probably want to verify that with @samccann). \`\`collections_paths\`\` would be represented as code and just look like one of the ini defaults fields, even though there are multiple ways to configure.

Reply:
I did a test doc build and confirmed this works as expected:

![image](https://user-images.githubusercontent.com/9920591/233459046-2baccf37-5605-495b-8c66-506f7f2968b9.png)

Reply:
lol cool cuz I'd have to do the same thing on this one :-)

Reply:
thanks for checking that!